sites/website/src/docs/api/fast-foundation.api.json

```diff
@@ -40,7 +40,11 @@
             {
               "kind": "Property",
               "canonicalReference": "@microsoft/fast-foundation!Accordion#expandmode:member",
+<<<<<<< HEAD
```

Comment:
merge conflicts?

Reply:
I'll generate this again - it's an automated thing so this will get resolved

Reply:
@awentzel, should be fixed.